pkg/github/teamreadwriter.go

```diff
@@ -335,6 +314,42 @@ func (g *TeamReadWriter) githubClientForOrg(ctx context.Context, orgID int64) (*
 	return g.client.WithAuthToken(token), nil
 }
 
+func (g *TeamReadWriter) addUserToTeam(ctx context.Context, client *github.Client, orgID, teamID int64, userID string) error {
+	membershipOpt := &github.TeamAddTeamMembershipOptions{Role: "member"}
+	if _, _, err := client.Teams.AddTeamMembershipByID(ctx, orgID, teamID, userID, membershipOpt); err != nil {
+		return fmt.Errorf("failed to add GitHub user(%s) for team(%d): %w", userID, teamID, err)
+	}
+	return nil
+}
+
+func (g *TeamReadWriter) addSubTeamToTeam(ctx context.Context, client *github.Client, orgID, teamID int64, groupID string) error {
+	childOrgID, childTeamID, err := parseID(groupID)
+	if err != nil {
+		return fmt.Errorf("could not parse group ID %s: %w", groupID, err)
+	}
+	if childOrgID != orgID {
+		return fmt.Errorf("cannot add team from another org as a child team")
+	}
```

Comment:
You can ignore this if you think it's unnecessary.

Feels like this is used in both `addSubTeamToTeam` and `removeSubTeamFromTeam`.

What do you think about we create a function `validateGroupID`, and at `pkg/github/teamreadwriter.go`, we can do 
```
if err := validateGroupID(); err != nil {xxxx}
if err := addSubTeam(ctx, client, orgID, teamID, childTeamID); err != nil {xxxx}
```